app/assets/stylesheets/_settings.scss

```diff
@@ -25,6 +25,8 @@ $further-info-icon-width: 20px;
 $further-info-icon-height: 20px;
 $further-info-background-color: $color-blue-medium;
 
+$keyword-dividing-color: #F0F4F4;
```

Comment:
Really this should be referencing a variable in the palette somewhere to encourage re-use. Is this definitely a new colour? If so you can just name it with http://chir.ag/projects/name-that-color/#F0F4F4 and add it further up.